flink-dist/pom.xml

```diff
@@ -121,6 +121,12 @@ under the License.
 			<version>${project.version}</version>
 		</dependency>
 
+		<dependency>
```

Comment:
I think we should also add this to pom of `flink-doc` to enable generating docs from options in `flink-kubernetes` module.

Reply:
`flink-kubernetes` module will be added to `flink-doc` pom in next commit when introduce the config options.

Reply:
Ok. That looks good to me.